app/backend/src/data/dummy_users.json

```diff
@@ -19,17 +19,15 @@
         "month": 3,
         "day": 2
       },
-      "languages": [["eng", "fluent"], ["fin", "fluent"]],
+      "languages": [
+        ["eng", "fluent"],
+        ["fin", "fluent"]
+      ],
       "occupation": "Mathematician",
       "about_me": "Some generic stuff.",
```

Comment:
Sorry these are all just formatting as I peeked in the file while considering adding a dummy female user for testing haha.